content/docs/command-reference/run.md

```diff
@@ -341,7 +341,7 @@ $ cd more_stages/
 $ dvc run -n process_data \
           -d data.in \
           -o result.out \
-          ./my_script.sh data.in result.out
+          ./my_script.sh --in data.in --out result.out
```

Comment:
why do we need `--in`, `--out`? Overall looks like a lot of very cosmetic changes to the (valid) commands in this PR

Reply:
The idea behind these `-d` reviews is to try keeping the dep file name in the `cmd` only when it's obvious why it's being passed to the script/program. Or to make it obvious, thus --in and --out. This is per #1700

But I don't have a strong opinion at all.

Reply:
> The idea behind these -d reviews 

it doesn't sound like a priority for now. Also. I would say it's rare to have explicit arguments like `--in` for your scripts. It might even be implicit completely, file internally reads it w/o getting it from CLI. Let's minimize these changes please.

Reply:
I guess the .in and .out file extensions were enough to make it self-documenting in this case... But what about the other example in line 382? Curr. `Rscript parsingxml.R --in data/Posts.xml --out data/Posts.csv`

Reply:
> doesn't sound like a priority for now

I hear you. I'm following the plan layed out in https://github.com/iterative/dvc.org/issues/2026 which starts by wrapping up pendine 1.0 updates.

Reply:
> Let's minimize these changes please.

OK sure, rolled back the 2 lines with --in/out.

Reply:
push changes?

Reply:
`Everything up-to-date` these changes are already gone from https://github.com/iterative/dvc.org/pull/2075/files#diff-f562129dd43887e87c126dd583630aeb0e838bf0993599f4a6a2520982a87048

Reply:
👍